apps/alb_monitor/test/alb_monitor/monitor_test.exs

```diff
@@ -11,9 +11,11 @@ defmodule ALBMonitor.MonitorTest do
     mock_instance_ip("10.0.0.2")
     mock_health_response([{"10.0.0.1", "healthy"}, {"10.0.0.2", "draining"}])
 
-    start!()
+    monitor_pid = start!()
 
     assert_receive :shutdown
+
+    refute Process.alive?(monitor_pid)
```

Comment:
question: wouldn't this pass even if the process has crashed instead of exiting normally?

Reply:
I was surprised, but in fact the process was still alive when I tested this locally.